src/__stories__/config.js

```diff
@@ -3,7 +3,7 @@ const { configure } = require("@kadira/storybook")
 function loadStories() {
   require("../components/__stories__/artwork")
   require("../components/__stories__/artwork_grid")
-  require("../components/__stories__/login_header")
+  require("../components/__stories__/modal_header")
```

Comment:
This is unrelated, but I wonder if there is any way to automatically load all the stories in `../components/__stories__/`.